src/Player.as

```diff
@@ -412,6 +425,10 @@ package {
       }
     }
 
+    private function onFrameReady(event:ClientEvent):void {
+      callAPI("frame:ready", event.data.timestamp);
```

Comment:
This should use ClientEvent.FRAME_READY constant, rather than reiterating the string.